CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.1.7
+
+- Changed MediaType parameter of StackStorm to access through the API endpoints instead of accessing directly to each service processes.
```

Comment:
Should probably add a note in the CHANGELOG about the new parameters for st2_dispatch.py since it changed from --st2-host to --st2-api-url and --st2-auth-url